distribution/scripts/ballerina/content_based_routing/main.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/data.jsondata;
+import ballerina/http;
+
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+listener http:Listener securedEP = new (9090,
+    secureSocket = {
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+final http:Client nettyEP = check new ("https://netty:8688",
+    secureSocket = {
+        cert: {
+            path: epKeyPath,
+            password: epKeyPassword
+        },
+        verifyHostName: false
+    }
+);
+
+service /contentBasedRouting on securedEP {
+    resource function post .(@http:Payload json data) returns json|error {
```

Comment:
Do we need this payload annotation?

Reply:
I think we dont this payload annotation, if we know the structure of the payload, we can use a record